Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -19,21 +19,16 @@ public class AudioManager : MonoBehaviour
   #region UnityMethods
   void Awake()
   {
-    if (instance != null)
```

Comment:
this led to all the sounds not being renewed after scene is reloaded

Reply:
Hmmm, interesting, just tried your changes and they work fine. 👍 

Can you please remove those empty lines in lines 22, 30 and 31?

Reply:
sure